packages/react-components/react-migration-v0-v9/etc/react-migration-v0-v9.api.md

```diff
@@ -51,6 +51,12 @@ export const slider: {
     fluid: () => GriffelStyle;
 };
 
+// @public (undocumented)
+export const spinner: {
+    inline: () => GriffelStyle;
+    v0SpinnerLabelStyle: () => GriffelStyle;
```

Comment:
v0 returns a different type, it should be `ICSSInJSStyle` from `@fluentui/react-northstar`